docs/setup/_setup_main.md

```diff
@@ -265,20 +265,20 @@ By default, the Firebase Flutter SDK auto-injects the Firebase JavaScript SDK wh
 
 2.  Load the script manually using one of the following alternatives:
 
-    - Add the SDK explicitly to your `web/index.html` file:
+    - Add the SDK explicitly to your `web/index.html` file, inside the `window.addEventListener` callback:
```

Comment:
what event? load? can you include the addEventListener call in the examples?

Reply:
I've added more context, let me know if it looks okay for you